Sources/TSCUtility/Platform.swift

```diff
@@ -49,19 +49,21 @@ public enum Platform: Equatable {
     }
 
     internal static func findCurrentPlatformLinux(_ fileSystem: FileSystem) -> Platform? {
-        if fileSystem.isFile(AbsolutePath("/etc/debian_version")) {
-            return .linux(.debian)
-        }
-        if fileSystem.isFile(AbsolutePath("/system/bin/toolbox")) ||
-            fileSystem.isFile(AbsolutePath("/system/bin/toybox")) {
-            return .android
-        }
-        if fileSystem.isFile(AbsolutePath("/etc/redhat-release")) ||
-            fileSystem.isFile(AbsolutePath("/etc/centos-release")) ||
-            fileSystem.isFile(AbsolutePath("/etc/fedora-release")) ||
-            Platform.isAmazonLinux2(fileSystem) {
-            return .linux(.fedora)
-        }
+        do {
+            if try fileSystem.isFile(AbsolutePath("/etc/debian_version")) {
```

Comment:
should we have a non-throwing initializer for static string?

Reply:
I'd like to keep that to tests since it is pretty rare for us to construct paths from static strings.